app/models/dynamic_router.rb

```diff
@@ -0,0 +1,14 @@
+class DynamicRouter < ApplicationRecord
```

Comment:
That could be a service?

Reply:
Or we need this to be persisted model in DB?

Reply:
I guess it could, it'd be more verbose.. let me check

Reply:
I mean it could be just moved to `services` and not inherit after `ApplicationRecord` if we don't need the model and the table